lib/pattern-visitor.js

```diff
@@ -21,16 +21,15 @@
   (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
   THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
-"use strict";
 
 /* eslint-disable no-undefined */
 
-const Syntax = require("estraverse").Syntax;
-const esrecurse = require("esrecurse");
+import { Syntax } from "estraverse";
```

Comment:
`estraverse` provides only cjs export. I think we should use only import default from `estraverse` because [interoperability with CommonJS](https://nodejs.org/api/esm.html#esm_interoperability_with_commonjs) doesn't guarantee named exports:

> When importing CommonJS modules, the module.exports object is provided as the default export. Named exports may be available, provided by static analysis as a convenience for better ecosystem compatibility.

The same for `esrecurse` and `@typescript-eslint/parser`.

Reply:
Ok, I've switched to the default import syntax for these CJS modules.